ERROR - 2023-06-12 02:28:00 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'lab_ci4', 3306, '', 0)
#1 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\app\Controllers\Artikel.php(32): CodeIgniter\BaseModel->findAll()
#7 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\CodeIgniter.php(934): App\Controllers\Artikel->admin_index()
#8 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Artikel))
#9 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\Commands\Server\rewrite.php(47): require_once('C:\\Users\\alifd\\...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\app\Controllers\Artikel.php(32): CodeIgniter\BaseModel->findAll()
#6 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\CodeIgniter.php(934): App\Controllers\Artikel->admin_index()
#7 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Artikel))
#8 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\Commands\Server\rewrite.php(47): require_once('C:\\Users\\alifd\\...')
#11 {main}
CRITICAL - 2023-06-12 02:28:00 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `artikel`', [], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Artikel.php(32): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Artikel->admin_index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Artikel))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\Users\\alifd\\OneDrive\\Documents\\xampp\\htdocs\\lab7_php_ci\\ci4\\public\\index.php')
ERROR - 2023-06-12 02:29:08 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'lab_ci4', 3306, '', 0)
#1 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\app\Controllers\Artikel.php(32): CodeIgniter\BaseModel->findAll()
#7 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\CodeIgniter.php(934): App\Controllers\Artikel->admin_index()
#8 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Artikel))
#9 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\Commands\Server\rewrite.php(47): require_once('C:\\Users\\alifd\\...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\app\Controllers\Artikel.php(32): CodeIgniter\BaseModel->findAll()
#6 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\CodeIgniter.php(934): App\Controllers\Artikel->admin_index()
#7 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Artikel))
#8 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\Commands\Server\rewrite.php(47): require_once('C:\\Users\\alifd\\...')
#11 {main}
CRITICAL - 2023-06-12 02:29:08 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `artikel`', [], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Artikel.php(32): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Artikel->admin_index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Artikel))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\Users\\alifd\\OneDrive\\Documents\\xampp\\htdocs\\lab7_php_ci\\ci4\\public\\index.php')
ERROR - 2023-06-12 02:29:52 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'lab_ci4', 3306, '', 0)
#1 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\app\Controllers\Artikel.php(12): CodeIgniter\BaseModel->findAll()
#7 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\CodeIgniter.php(934): App\Controllers\Artikel->index()
#8 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Artikel))
#9 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\Commands\Server\rewrite.php(47): require_once('C:\\Users\\alifd\\...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\app\Controllers\Artikel.php(12): CodeIgniter\BaseModel->findAll()
#6 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\CodeIgniter.php(934): App\Controllers\Artikel->index()
#7 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Artikel))
#8 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\Commands\Server\rewrite.php(47): require_once('C:\\Users\\alifd\\...')
#11 {main}
CRITICAL - 2023-06-12 02:29:52 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `artikel`', [], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Artikel.php(12): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Artikel->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Artikel))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\Users\\alifd\\OneDrive\\Documents\\xampp\\htdocs\\lab7_php_ci\\ci4\\public\\index.php')
ERROR - 2023-06-12 02:31:38 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'lab_ci4', 3306, '', 0)
#1 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\app\Controllers\Artikel.php(32): CodeIgniter\BaseModel->findAll()
#7 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\CodeIgniter.php(934): App\Controllers\Artikel->admin_index()
#8 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Artikel))
#9 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\Commands\Server\rewrite.php(47): require_once('C:\\Users\\alifd\\...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\app\Controllers\Artikel.php(32): CodeIgniter\BaseModel->findAll()
#6 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\CodeIgniter.php(934): App\Controllers\Artikel->admin_index()
#7 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Artikel))
#8 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\Commands\Server\rewrite.php(47): require_once('C:\\Users\\alifd\\...')
#11 {main}
CRITICAL - 2023-06-12 02:31:38 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `artikel`', [], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Artikel.php(32): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Artikel->admin_index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Artikel))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\Users\\alifd\\OneDrive\\Documents\\xampp\\htdocs\\lab7_php_ci\\ci4\\public\\index.php')
ERROR - 2023-06-12 02:35:24 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'lab_ci4', 3306, '', 0)
#1 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\app\Controllers\Artikel.php(32): CodeIgniter\BaseModel->findAll()
#7 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\CodeIgniter.php(934): App\Controllers\Artikel->admin_index()
#8 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Artikel))
#9 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\Commands\Server\rewrite.php(47): require_once('C:\\Users\\alifd\\...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\app\Controllers\Artikel.php(32): CodeIgniter\BaseModel->findAll()
#6 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\CodeIgniter.php(934): App\Controllers\Artikel->admin_index()
#7 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Artikel))
#8 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\Users\alifd\OneDrive\Documents\xampp\htdocs\lab7_php_ci\ci4\system\Commands\Server\rewrite.php(47): require_once('C:\\Users\\alifd\\...')
#11 {main}
CRITICAL - 2023-06-12 02:35:24 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `artikel`', [], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Artikel.php(32): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Artikel->admin_index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Artikel))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\Users\\alifd\\OneDrive\\Documents\\xampp\\htdocs\\lab7_php_ci\\ci4\\public\\index.php')
INFO - 2023-06-12 02:50:56 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2023-06-12 03:02:20 --> Class "Config\App\Filters\Auth" not found
in SYSTEMPATH\Filters\Filters.php on line 166.
 1 SYSTEMPATH\CodeIgniter.php(473): CodeIgniter\Filters\Filters->run('admin/artikel', 'before')
 2 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 3 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 4 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\Users\\alifd\\OneDrive\\Documents\\xampp\\htdocs\\lab7_php_ci\\ci4\\public\\index.php')
CRITICAL - 2023-06-12 03:05:44 --> Class "Config\App\Filters\Auth" not found
in SYSTEMPATH\Filters\Filters.php on line 166.
 1 SYSTEMPATH\CodeIgniter.php(473): CodeIgniter\Filters\Filters->run('admin/artikel', 'before')
 2 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 3 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 4 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\Users\\alifd\\OneDrive\\Documents\\xampp\\htdocs\\lab7_php_ci\\ci4\\public\\index.php')
CRITICAL - 2023-06-12 03:31:35 --> Class "Config\App\Filters\Auth" not found
in SYSTEMPATH\Filters\Filters.php on line 166.
 1 SYSTEMPATH\CodeIgniter.php(473): CodeIgniter\Filters\Filters->run('admin/artikel', 'before')
 2 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 3 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 4 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\Users\\alifd\\OneDrive\\Documents\\xampp\\htdocs\\lab7_php_ci\\ci4\\public\\index.php')
